learning_assistant/utils.py

```diff
@@ -143,7 +143,7 @@ def get_optimizely_variation(user_id, enrollment_mode):
         variation_key = None
     else:
         optimizely_client = optimizely.Optimizely(sdk_key=settings.OPTIMIZELY_FULLSTACK_SDK_KEY)
-        user = optimizely_client.create_user_context(str(user_id), {'enrollment_mode': enrollment_mode})
+        user = optimizely_client.create_user_context(str(user_id), {'lms_enrollment_mode': enrollment_mode})
```

Comment:
This was changed to match Alie's changes in https://github.com/edx/frontend-lib-learning-assistant/pull/85.